src/views/ProjectView.vue

```diff
@@ -146,12 +185,12 @@ onMounted(() => {
 
       <div v-else>
         <ImageCarousel
-          v-if="imageDisplayToggle && store.state.smallImageCache.length && store.state.largeImageCache.length"
-          :data="store.state.smallImageCache"
+          v-if="imageDisplayToggle && selectedProjectImages.length"
```

Comment:
So if you aren't part of any projects, nothing will show up I guess. We should have a third option that says "Please create a project to use datalab" or something like that